### ISSUE TITLE
No validation of District when adding/editing user accounts.

### ISSUE THREAD

**Issue by gmcressman** [ICTatRTI/coconut-analytics#311]

**Comment by scloo** [comment]
Perhaps adding a type ahead selection of districts like in the Mobile might be helpful.

**Comment by gmcressman** [comment]
Yes, it would. I also looked at the (actual) historical data in Coconut Surveillance this morning. I found the following:

- The database contains multiple forms of the same district name (e.g. "CHAKE CHAKE" and "CHAKECHAKE". This indicates that, at least for a time, there has been no validation of district names.
- There same facility may have been identified as being in several different shehia. I found cases in which the same facility name appears in as many as 11 different shehia. 

I believe I can use DHIS2 to generate a report to see whether two or more facilities in a district have the same facility name.  I suspect that the lack of validation of district, shehia, and facility names, and validation of the organizational relationships, has created a mess.

**Comment by scloo** [comment]
I have added a dropdown Select for the District field. As well as for the Facilities view namely the Region and Districts.

In the User View, I noticed that some of the user's assigned district does not appear in the District list in the Hierarchy. For example CENTRAL, URBAN, NORTH A, NORTH B,  SOUTH, WEST, etc. These definitely need to be cleaned up. In any case, when editing a user, if the District is not found in the Hierarchy list, it will be displayed as blank.
